elide-core/src/test/java/com/yahoo/elide/graphql/ModelBuilderTest.java

```diff
@@ -33,6 +33,7 @@
     private EntityDictionary dictionary;
 
     private static final String ID = "id";
```

Comment:
Is this still in use? It appears as though `idArgument` has been renamed to `ids` everywhere. Is there a singular `id` remaining?